jerry-core/parser/js/js-lexer.c

```diff
@@ -2264,13 +2264,13 @@ lexer_expect_identifier (parser_context_t *context_p, /**< context */
  */
 void
 lexer_expect_object_literal_id (parser_context_t *context_p, /**< context */
-                                bool must_be_identifier) /**< only identifiers are accepted */
+                                uint32_t ident_opts) /**< lexer_obj_ident_opts_t option bits */
 {
   lexer_skip_spaces (context_p);
 
 #ifndef CONFIG_DISABLE_ES2015_CLASS
-  int is_class_method = ((context_p->status_flags & PARSER_IS_CLASS)
```

Comment:
Is this flag still used?

Reply:
It's no longer needed so it had been removed [here](https://github.com/jerryscript-project/jerryscript/pull/2436/files/2445998bf708053668fb6ee15fb7d6fa2fd4fd57#diff-e5e614e4a1fa8a1b485bec8873dff088L71).